dxr/templates/nunjucks/breadcrumbs.html

```diff
@@ -0,0 +1,11 @@
+{% macro breadcrumbs(www_root, paths_and_names) %}
+  <div class="breadcrumbs">
+    {%- for path, name in paths_and_names -%}
+      {%- if (not loop.first) or (loop.last and loop.length != 1) -%}
```

Comment:
Hah, good explanation! I guess I didn't really think about simplifying the expression, I just looked up the precedence rules and unpacked them...